docs/guides/scripts.md

```diff
@@ -40,20 +40,22 @@ drwxr-xr-x    3 fzeoli  staff      96 Jul 30 15:27 scripts
 drwxr-xr-x    3 fzeoli  staff      96 Jul 30 15:27 test
 ```
 
-Inside `scripts/` you will find `sample-script.js`:
-```js
+Inside `scripts/` you will find `sample-script.js`. Add the highlighted lines to it:
+
+```js{1-2,5}
 const bre = require("@nomiclabs/buidler");
+const ethers = bre.ethers;
 
 async function main() {
-  // You can run Buidler tasks from a script.
-  // For example, we make sure everything is compiled by running "compile"
   await bre.run("compile");
```

Comment:
What makes the script standalone? There are 3 lines being highlighted, am I missing something?

Reply:
Oh, I think I see your point here. It reads as those lines are responsible for making the script standalone, not in fact that the BRE is being extended. I'll fix it by adding some extra information/comments.

Reply:
yeah, but compile runs anyway, so that line doesn't make it standalone

Reply:
Below the script:

> By accessing the [Buidler Runtime Environment](#) at the top, you are now allowed to run the script in a standalone fashion. Lines 2 and 7 are just examples on how to use it. Let's re-run the script with `node`:

Reply:
Okay, I was misunderstanding everything. Forget my initial comment here and let's just explain how these highlighted lines make it standalone after the code snipper. Basically explain that:

- the BRE components are accessed explicitly through the BRE rather than using from a global scope injection that's only available when running through Buidler
- the compile task is invoked manually. When Buidler runs a script through `buidler run` compile is run before, so to make sure everything is compiled when running directly through node.js we call `compile`